django_q/cluster.py

```diff
@@ -691,6 +691,8 @@ def scheduler(broker: Broker = None):
                     if type(args) != tuple:
                         args = (args,)
                 q_options = kwargs.get("q_options", {})
+                if s.intended_date_kwarg:
```

Comment:
It would be nice if we have a default here and apply it to all schedules, say "intended_date", so we don't necessarily have to set it. If we would leave it like this, then existing apps would likely have to do another migration for all schedules to update them with the new kwarg and I feel like most people would go with a default anyway. 
Not a position a hold strongly though. Happy to merge without.

Reply:
The danger I see in having a default is that it may collide - even if probability is low - with an existing kwarg in the called function. This could have unexpected effects that could be difficult to debug.
Also if a user wants to use this feature he will, most likely, have to adapt his task functions. Having to change the Schedule configuration is an additional burden, but a limited one in comparison I believe.
Could be interesting to get a third opinion on this.

Reply:
That's fair. Thanks.